package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
-    "prepublish": "npm run clean && npm run build",
```

Comment:
`prepublish` was removed, because you don't want to make any changes to checked in source files before publishing. You just commit, tag, push, and publish like normal.